run.py

```diff
@@ -8,31 +9,35 @@
 from library.evaluation import classification_metrics
 from library.examples.all_transformations import all_transformations
 
-from plugins import WandbPlugin
+from blocks.pipeline import Pipeline
+from typing import Tuple
 
+hate_speech_data = load_data("data/original", preprocess_config)
 
-train_dataset, test_dataset = load_data("data/original", preprocess_config)
 
-pipeline = hate_speech_detection_pipeline()
+def run(pipeline: Pipeline, data: Tuple[Dataset, Dataset] = hate_speech_data) -> None:
```

Comment:
hmm maybe no default parameter here? would that ve ok? it's probably better to be explicit about it!:)

Reply:
Makes sense!